core/dslmcode/shared/drupal-7.x/modules/elmsln_contrib/cis_connector/modules/features/cis_section/cis_section.views_default.inc

```diff
@@ -190,107 +194,6 @@ function cis_section_views_default_views() {
   $handler->display->display_options['menu']['name'] = 'menu-elmsln-settings';
   $handler->display->display_options['menu']['context'] = 0;
   $handler->display->display_options['menu']['context_only_inline'] = 0;
-  $export['cis_section_list'] = $view;
```

Comment:
Good catch - cis/user-list was never there. I discovered this when I did the export of user-list view and I couldn't find the right file to put it's feature in. I accidently copied some of it's code in here so I'll have to go revert it to what it was and re-export.